test/bin/ci_phase_iso_build.sh

```diff
@@ -124,6 +124,28 @@ cd "${ROOTDIR}/test/"
 # shellcheck source=test/bin/common.sh
 source "${SCRIPTDIR}/common.sh"
 
+# replace the images to OKD
+okd_url=quay.io/okd/scos-release
+okd_releaseTag=4.17.0-0.okd-scos-2024-08-21-100712
+
+oc adm release info ${okd_url}:${okd_releaseTag} >all_images
+
+for op in $(cat ${ROOTDIR}/assets/release/release-x86_64.json  | jq -e -r  '.images | keys []') 
+do
+    image=$(oc adm release info --image-for=${op} ${okd_url}:${okd_releaseTag}) 
+    if [ $? -eq 0 ] ; then
+        echo "${op} ${image}"
+        jq --arg a "${op}" --arg b "${image}"  '.images[$a] = $b' assets/release/release-x86_64.json >/tmp/release-x86_64.json.tmp
```

Comment:
Should you concat lines here using `>>`?

Reply:
its not concatenating, its replacing each time one image and overriding the file.

Reply:
see for example:
```
+ 10:57:33.558234231 ./test/bin/ci_phase_iso_build.sh:138 	jq --arg a cli --arg b quay.io/okd/scos-content@sha256:580848455b5c4f5a28b2dc4fa32fa11ead577f02e5649b660491ba7bab31542b '.images[$a] = $b' ../assets/release/release-x86_64.json
+ 10:57:33.562627453 ./test/bin/ci_phase_iso_build.sh:139 	mv /tmp/release-x86_64.json.tmp ../assets/release/release-x86_64.json
```

https://prow.ci.openshift.org/log?container=test&id=1829467008731713536&job=pull-ci-openshift-microshift-release-4.17-microshift-metal-tests